### PR TITLE
Clean up code for actions in <amp-audio>

### PR DESCRIPTION
**Pull request by nainar**
As per suggestions made by @alanorozco  in #15207

### CODE REVIEW

**Comment by nainar** [comment]
@alanorozco please take a look?

File: extensions/amp-audio/0.1/amp-audio.js [hunk 1]
```diff
@@ -46,8 +46,8 @@ export class AmpAudio extends AMP.BaseElement {
     /** @private {!../../../src/mediasession-helper.MetadataDef} */
     this.metadata_ = EMPTY_METADATA;
 
-    /** @private {boolean} */
-    this.isPlaying_ = false;
+    /** @public {boolean} */
```
[the remark this answers is not in the document]
Comment (nainar) [reply extensions/amp-audio/0.1/amp-audio.js]: Done.

File: extensions/amp-audio/0.1/amp-audio.js [hunk 2]
```diff
@@ -119,53 +119,65 @@ export class AmpAudio extends AMP.BaseElement {
...
   }
 
   /**
    * Returns whether the audio is playing or not.
    * @returns {boolean}
    * @VisibleForTesting
    */
-  playerStatus() {
-    return this.isPlaying_;
+  setPlayingStateForTesting_(val) {
```
[the remark this answers is not in the document]
Comment (nainar) [reply extensions/amp-audio/0.1/amp-audio.js]: Done.

File: extensions/amp-audio/0.1/amp-audio.js [hunk 3]
```diff
@@ -119,53 +119,65 @@ export class AmpAudio extends AMP.BaseElement {
...
+    }
   }
 
+  /**
+   * Returns whether `<amp-audio>` has an `<amp-story>` for an ancestor.
+   * @returns {boolean}
+   * @VisibleForTesting
+   */
+  isStoryDescendant_() {
+    return closestByTag(this.element, 'AMP-STORY') ? true : false;
```
[the remark this answers is not in the document]
Comment (nainar) [reply extensions/amp-audio/0.1/amp-audio.js]: Done.

**Comment by nainar** [comment]
@kristoferbaxter per offline discussion could you take a look at the changes to reduce the size?

File: extensions/amp-audio/0.1/amp-audio.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (kristoferbaxter) [reply extensions/amp-audio/0.1/amp-audio.js]: Moved back to public since the linter complained about this. Let's address in a separate PR.

**Comment by kristoferbaxter** [comment]
LGTM

**Pull request merged**